test/unit/text-fragment-utils-test.js

```diff
@@ -1,16 +1,5 @@
-import * as utils from '../src/text-fragment-utils.js';
-
-// A normalizing helper function to try and compare extracted marks in a way
-// that's robust-ish to reformatting/prettifying of the HTML input files.
-const marksArrayToString = (marks) => {
-  // Extract text content with normalized whitespace
-  const text = marks.map(
-      (node) => node.textContent.replace(/[\t\n\r ]+/g, ' '),
-  );
-  // Join, re-normalize (because block elements can add extra whitespace if
-  // HTML contains newlines), and trim.
-  return text.join('').replace(/[\t\n\r ]+/g, ' ').trim();
-};
+import * as utils from '../../src/text-fragment-utils.js';
+import {marksArrayToString} from '../utils/marksArrayToString.js';
```

Comment:
Format nit: are there usually spaces around the curly braces, e.g. `{ marksArrayToString }`?

Reply:
👍